components/editor/prosemirror-schema/create-new-schema.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Create a New Schema
+page_title:  Create a New Schema
+description: Expore how to create a new schema for the Editor for Blazor.
+slug: editor-create-new-schema
+tags: telerik,blazor,editor,prosemirror,new,schema
+published: True
+position: 5
+---
+
+# Create a New Schema
+
+This article describes how you can create a new [ProseMirror schema]({%slug editor-prosemirror-schema-overview%}) for the Editor to use. Creating a new schema is useful if you want to change the majority of the default schema.
+
+@[template](/_contentTemplates/editor/general.md#prosemirror-schema-prerequisites)
+
+## Basics
+
+@[template](/_contentTemplates/editor/general.md#prosemirror-schema-general-info)
+
+
+## Creating a New Schema
+
+1. In the global app scope (the `window` object) declare a JS function that returns an instance of the [ProseMirror `Schema` class](https://prosemirror.net/docs/ref/#model.Schema).
+1. Create a new instance of the `Schema` object and include your desired nodes and marks in it.
+1. Return the new `Schema` object.
+1. Pass the name of the JS function to the `Schema` parameter of the Editor.
+
+> Some of the ProseMirror plugins that the Editor uses by design depend on specific nodes in the default ProseMirror schema of the Editor. When creating a new schema from scratch, it is possible to get an exception if you do not include the needed nodes in your custom schema. In this case, you can either include the corresponding nodes or pass and a [custom empty collection of plugins to the Editor]({%slug editor-prosemirror-plugins%}) to override the built-in ones.
```

Comment:
Check this segment out - it sounds odd. Perhaps an extra word/phrase, or a missing one is causing the issue.

> In this case, you can either include the corresponding nodes **or pass and a custom empty collection** of plugins to the Editor